Purchases/RCAttributionFetcher.m

```diff
@@ -46,7 +46,7 @@ - (nullable NSString *)advertisingIdentifier
 
 - (nullable NSString *)identifierForVendor
 {
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IOS
```

Comment:
what about tv os? because TARGET_OS_IPHONE also includes Apple TVs right?

Reply:
world's most confusing macros (the tv is iPhone but it isn't iOS?). Good catch, I'll update